source/documentation/adrs/adr-006.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: ADR-006 Migrate to Github Actions instead of CircleCI
+last_reviewed_on: 2022-02-09
+review_in: 12 months
+---
+
+# <%= current_page.data.title %>
+
+### Date: 14-02-2023
+
+## Status
+
+✅ Accepted
+
+## Context
+
```

Comment:
Worth adding the background of using Circle since 2018 and how we trialled and prefered GH actions on smaller projects initially. Feels a bit focused on recent security incidents over longer term usage issues. Might also be worth mentioning GitHub Actions being better integrated into the core tooling i.e. GitHub

Reply:
I've change the context around to be more about gh positives rather than circle's negatives